source/mod_alloc.f90

```diff
@@ -189,20 +187,6 @@ subroutine print_alloc(ename, type, requested_bits)
 
 end subroutine print_alloc
 
-subroutine alloc_exit
-#ifdef DEBUG
-  use mod_units
-  integer memunit
-  call f_requestUnit("maximum_memory_allocation_mbytes.txt",memunit)
-  call f_open(unit=memunit,file="maximum_memory_allocation_mbytes.txt",formatted=.true.,mode="w")
-  write(memunit,"(f10.3)") real(maximum_bits,real64)/real(mbyte,real64)
-  flush(memunit)
-  call f_close(memunit)
-#else
-  return
-#endif
-end subroutine alloc_exit
-
```

Comment:
Is the max memory usage saved somewhere else?

Reply:
Yes, in the `sim_meta.dat` file.